test/Microsoft.Net.Http.Headers.Tests/CookieHeaderValueTest.cs

```diff
@@ -154,7 +154,7 @@ public void CookieHeaderValue_Ctor2InitializesCorrectly(string name, string valu
         public void CookieHeaderValue_Value()
         {
             var cookie = new CookieHeaderValue("name");
-            Assert.Equal(String.Empty, cookie.Value);
+            Assert.Equal(string.Empty, cookie.Value);
```

Comment:
Is that what's intended here? IMO using the IEnumerable overload when checking strings isn't the most intuitive.